app/controllers/declaration/ModelCacheable.scala

```diff
@@ -18,9 +18,11 @@ package controllers.declaration
 
 import models.ExportsDeclaration
 import models.requests.JourneyRequest
+import services.audit.{AuditService, AuditTypes}
 import services.cache.ExportsCacheService
 import uk.gov.hmrc.http.HeaderCarrier
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Is this really required?

Reply:
Yeah, pr-bot slapped my wrist for this too! :slightly_frowning_face: 

Removed